rs/state_machine_tests/src/lib.rs

```diff
@@ -938,6 +940,8 @@ impl StateMachineBuilder {
             subnet_size: SMALL_APP_SUBNET_MAX_SIZE,
             nns_subnet_id: None,
             subnet_id: None,
+            max_stream_messages: None,
+            target_stream_size_bytes: None,
```

Comment:
I suppose this is simply keeping it consistent, but we also have arguments that are initialized to a default value (e.g. `subnet_size` above or `routing_table` below). Why not simply have the fields be of `usize` type and initializing them with the respective constants instead?

```suggestion
            max_stream_messages: MAX_STREAM_MESSAGES,
            target_stream_size_bytes: TARGET_STREAM_SIZE_BYTES,
```

Reply:
It was. It's now just a `usize`.